src/java/picard/analysis/directed/InsertSizeMetricsCollector.java

```diff
@@ -121,7 +133,13 @@ public void addMetricsToFile(final MetricsFile<InsertSizeMetrics,Integer> file)
                 final double total = Histogram.getCount();
```

Comment:
Would you mind fixing the `Histogram` variable name to `histogram` while you're in here please?